policy/release/labels.rego

```diff
@@ -146,15 +190,90 @@ deny contains result if {
 	result := lib.result_helper(rego.metadata.chain(), [error])
 }
 
-image_labels contains label if {
-	some name, value in input.image.config.Labels
-	count(value) > 0
-	label := {"name": name, "value": value}
+# METADATA
+# title: Inaccessible parent image manifest
+# description: >-
+#   The parent image manfest is not accessible.
+# custom:
+#   short_name: inaccessible_parent_manifest
+#   failure_msg: Manifest of the image %q, parent of image %q is inaccessible
+#   solution: >-
+#     Check the provided authentication configuration and the credentials
+#     within it.
+#   collections:
+#   - redhat
+#
+deny contains result if {
+	manifest := ec.oci.image_manifest(input.image.ref)
+
+	name := manifest.annotations["org.opencontainers.image.base.name"]
+	digest := manifest.annotations["org.opencontainers.image.base.digest"]
+	parent_ref = sprintf("%s@%s", [name, digest])
+
+	parent_manifest := ec.oci.image_manifest(parent_ref)
+	is_null(parent_manifest)
+	result := lib.result_helper(rego.metadata.chain(), [parent_ref, input.image.ref])
+}
+
+# METADATA
+# title: Inaccessible parent image config
+# description: >-
+#   The parent image config is not accessible.
+# custom:
+#   short_name: inaccessible_parent_config
+#   failure_msg: Image config of the image %q, parent of image %q is inaccessible
+#   solution: >-
+#     Check the provided authentication configuration and the credentials
+#     within it.
+#   collections:
+#   - redhat
+#
+deny contains result if {
+	manifest := ec.oci.image_manifest(input.image.ref)
+
+	parent_name := manifest.annotations["org.opencontainers.image.base.name"]
+	parent_digest := manifest.annotations["org.opencontainers.image.base.digest"]
+	parent_ref = sprintf("%s@%s", [parent_name, parent_digest])
```

Comment:
Could consider moving these into a parent_ref_from_manifest (or something) helper rule to reduce repetition.

Reply:
:+1: found a nice refactor that can be done here